src/windows/models/transition_model.py

```diff
@@ -65,10 +65,10 @@ def filterAcceptsRow(self, sourceRow, sourceParent):
 
     def lessThan(self, left, right):
         """Sort with both group name and transition name"""
-        leftData = self.sourceModel().data(left)
-        leftGroup = self.sourceModel().data(left, 257)  # Strange way to access 'type' column in model
-        rightData = self.sourceModel().data(right)
-        rightGroup = self.sourceModel().data(right, 257)
+        leftData = left.data(self.sortRole())  # Transition name (or other role data)
+        rightData = right.data(self.sortRole())
+        leftGroup = left.sibling(left.row(), 2).data()  # group column
```

Comment:
2 is not as cool as 257, but otherwise, looks good!

Reply:
I mean, if it was `42` I'd _absolutely_ have kept it.